sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryClient.java

```diff
@@ -36,51 +36,96 @@
 
 
 /**
- * HTTP-based client that interacts with Azure Schema Registry service to store and retrieve schemas on demand.
+ * {@link SchemaRegistryClient} is an HTTP-based client that interacts with Azure Schema Registry service to store and
+ * retrieve schemas on demand.
  *
- * <p><strong>Register a schema</strong></p>
- * Registering a schema returns a unique schema id that can be used to quickly associate payloads with that schema.
- * <!-- src_embed com.azure.data.schemaregistry.schemaregistryclient.registerschema -->
+ * <p><strong>Sample: Construct a {@link SchemaRegistryClient}</strong></p>
+ *
+ * <p>The following code sample demonstrates the creation of the synchronous client
+ * {@link com.azure.data.schemaregistry.SchemaRegistryClient}.  The credential used is {@code DefaultAzureCredential}
+ * because it combines commonly used credentials in deployment and development and chooses the credential to used based
+ * on its running environment.</p>
+ *
+ * <!-- src_embed com.azure.data.schemaregistry.schemaregistryclient.construct -->
+ * <pre>
+ * DefaultAzureCredential azureCredential = new DefaultAzureCredentialBuilder&#40;&#41;
+ *     .build&#40;&#41;;
+ * SchemaRegistryClient client = new SchemaRegistryClientBuilder&#40;&#41;
+ *     .fullyQualifiedNamespace&#40;&quot;https:&#47;&#47;&lt;your-schema-registry-endpoint&gt;.servicebus.windows.net&quot;&#41;
+ *     .credential&#40;azureCredential&#41;
+ *     .buildClient&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.data.schemaregistry.schemaregistryclient.construct -->
+ *
+ * <p><strong>Sample: Register a schema</strong></p>
+ *
+ * <p>The following code sample demonstrates registering an Avro schema.  The
+ * {@link com.azure.data.schemaregistry.models.SchemaProperties} returned contains the schema's id.  This id uniquely
+ * identifies the schema and can be used to quickly associate payloads with that schema.</p>
```

Comment:
We can mention more details about supported formats of schemas and also link out to MS docs which talks more about those schema definitions with sample structures.